lib/memoization.js

```diff
@@ -11,9 +11,7 @@ const MEMOIZED = new LRU({
   length: (entry, key) => {
     if (key.startsWith('key:')) {
       return entry.data.length
-    }
-
-    if (key.startsWith('digest:')) {
+    } else {
```

Comment:
Just to make sure we keep the intention clear, I'd do something like this:

```js
if (key.startsWith('key:')) {
  return entry.data.length
} else /* istanbul ignore else */ if (key.startsWith('digest:')) {
  return entry.length
} else {
  throw new Error('invalid memoization key name')
}
```

Reply:
Or if terseness is your thing, and you don't think it's worth the visual overhead:

```js
return key.startsWith('key:') ? entry.data.length : entry.length
```

Reply:
oh yeah, def prefer the shorter arrow-function version 😊 

one issue with the first example is that we would _also_ have to `/* istanbul ignore else */` the last `else` statement since we wouldn't be able to unit test it all the same 🤔 so it's still kind of the same